decide/lic8.cpp

```diff
@@ -0,0 +1,26 @@
+#include <decide/decide.hpp>
+#include <decide/lics.hpp>
+#include <decide/utils.hpp>
+
+/// There exists at least one set of three data points separated by exactly
+/// APTS and BPTS consecutive intervening points, respectively, that
+/// cannot be contained within or on a circle of radius RADIUS1. The
+/// condition is not met when NUMPOINTS < 5.
+/// Clarification: we have our point, then APTS points, then our next point,
+/// then BPTS, then our last point.
+///
+/// 1 ≤ APTS, 1 ≤ BPTS
+/// APTS + BPTS ≤ (NUMPOINTS − 3)
+bool lic8(Points points, Parameters parameters) {
+  int apts = parameters.APTS;
+  int bpts = parameters.BPTS;
+
```

Comment:
Do you take the `The condition is not met when NUMPOINTS < 5.` criteria into account here?

Reply:
Ah, I guess I could add some asserts as mentioned in another issue. Will fix.